unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Addressables/Tests/AddressablesValidationTests.cs

```diff
@@ -3,20 +3,23 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using UnityEditor;
 using UnityEditor.AddressableAssets;
 using UnityEditor.AddressableAssets.Build.AnalyzeRules;
 using UnityEngine;
 
+[Category("EditModeCI")]
+
 public class AddressablesValidationTests
 {
-    private static readonly string[] EXCLUDED_FILE_TYPES = { }; // "shader", "png", "jpg"
     private const string NO_ISSUES_FOUND = "No issues found";
+    private static readonly string[] EXCLUDED_FILE_TYPES = { }; // "shader", "png", "jpg"
 
-    [TestCase("Rendering")]
+    // [TestCase("Scripts/MainScripts/DCL/Controllers/HUD/NotificationHUD")]
```

Comment:
is there a reason of why the test case is left as a comment?

Reply:
It is a long story. Previously, each test was with the attribute  `Category("EditModeCI")` and this particular test wasn't included (because it was failing). Then, I move  `Category("EditModeCI")`  attribute to the class declaration, so I needed to disable this test to pass the CI. Anyway, I will re-enable it, because dependent commit was merged (in fact, now I have a conflict there exactly about this test) ;)